Task01_poem/Task01_poem/Program.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Task01_poem
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.ForegroundColor = ConsoleColor.Green;
+            
+            Console.WriteLine("Луна и смерть.");
+
+            Console.WriteLine();
+
+            Console.ForegroundColor = ConsoleColor.Yellow;
+            Console.WriteLine("Зубы кости слоновой\r\nу луны ущербленной.\r\nО, канун умиранья!\r\nНи былинки зеленой,\r\nопустелые гнезда,\r\nпересохшие русла…\r\nУмирать под луною\r\nтак старо и так грустно!");
```

Comment:
Если весь текст стихотворения помещать в один оператор (выводить одной строковой константой), символ `\r` возврата каретки лишний. Достаточно использовать перевод строки `\n`.
```suggestion
            Console.WriteLine("Зубы кости слоновой\nу луны ущербленной.\nО, канун умиранья!\nНи былинки зеленой,\nопустелые гнезда,\nпересохшие русла…\nУмирать под луною\nтак старо и так грустно!");
```